src/services/badges/badges.ts

```diff
@@ -191,4 +211,99 @@ export class BadgesService extends TheBadgeSDKConfig implements BadgesServiceMet
     // return badge image ipfs hash
     return badgeImageIPFSHash
   }
+
+  /**
+   * Mint a new badge of a certain model
+   *
+   * @param userAddress
+   * @param badgeModelId
+   * @param evidences
+   * @param base64PreviewImage
+   */
+  public async mint(
+    userAddress: string,
+    badgeModelId: string,
+    evidences: Record<number, unknown>, // { evidenceIndex: evidenceValue }, example: { 0: 'text1', 1: 'text2', 2: date1 }
```

Comment:
I think this comment explaining what's inside the evidences record should be above, near the @param comment

Reply:
fixed